### PR TITLE
feat: added an option to use double dropdown menu

### PR DESCRIPTION
**Pull request by shawn320**
# DS-##(Jira Issue)-A###(component ID)-(component name)

**Reminder: Add all changes for this PR to the CHANGELOG.md**

Main Reviewer: @numbap 

#### Description & Background Context:

Added an option to use double dropdown menu. The double dropdown has numbers only. User can set min and max value for both dropdowns. 

#### Where should the reviewer start?

File name: FormDropdown.js

#### How should this be tested?

Check the double dropdown menu on Storybook: http://localhost:6006/?path=/story/components-formdropdown--double-dropdown

### CODE REVIEW

**Review by numbap — CHANGES_REQUESTED** [review]
Screenshot 2023-06-16 at 2 13 49 PM
Screenshot 2023-06-16 at 2 14 00 PM


There are errors in the props.

File: src/components/CustomDropdown/CustomDropdown.js [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+          <FontAwesomeIcon
+            icon={faCaretDown}
+            style={{ color: "#666666" }}
+            className="ds-absolute ds-right-0 ds-px-[14px]"
+          />
+        </button>
+        <div className="dropdown-open">
+          {isOpen && (
+            <ul className="dropdown-options">
+              {combinedOptions
```
Comment (numbap) [inline src/components/CustomDropdown/CustomDropdown.js:74]: In future iterations, this should be a seperate function.

File: src/components/FormDropdown/FormDropdown.js [hunk 2]
```diff
@@ -55,104 +21,118 @@ export const FormDropdown = (props) => {
...
+    hasError: PropTypes.bool,
+    errorText: PropTypes.string,
+    minValue: PropTypes.number,
+    maxValue: PropTypes.number,
+    hasSearch: PropTypes.bool,
+  }),
+  /**
+   * Dropdown props for the second dropdown in case of doubleDropdown
+   */
+  dropdown2Props: PropTypes.shape({
```
Comment (numbap) [inline src/components/FormDropdown/FormDropdown.js:105]: Is 2 the maxunm number of Dropdowns that will ever exist in this component?

File: src/components/CustomDropdown/CustomDropdown.js (hunk 1 above)
Comment (shawn320) [reply src/components/CustomDropdown/CustomDropdown.js]: This is a good suggestion, thanks. Updated the code to separate the string and number options.

File: src/components/FormDropdown/FormDropdown.js (hunk 2 above)
Comment (shawn320) [reply src/components/FormDropdown/FormDropdown.js:109]: The double dropdown is built specifically for the OAS estimator team, other people might use this pattern in the future, but for now, it's made for OAS team's need. 
If user want more dropdowns, they can just add more single dropdowns to their app.

**Review by numbap — APPROVED** [review]

**Pull request merged**